gym_sts/autoencoder/train.py

```diff
@@ -234,15 +262,27 @@ def train():
                 results=results, prefix=f"Batch: {batch_num+1}/{total_batches}"
             )
 
+            to_log = dict(
+                loss=results["loss"].double(),
+                top1=results["top1"].double(),
+                epoch=epoch + batch_num / total_batches,
```

Comment:
Is this meant to be a float value? halfway through epoch 1, the value should be 1.5?

Reply:
Yes that's right.